pkgs/tools/misc/parallel/default.nix

```
@@ -11,7 +11,8 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "man" "doc" ];
 
-  nativeBuildInputs = [ makeWrapper ];
+  strictDeps = true;
+  nativeBuildInputs = [ makeWrapper perl ];
   buildInputs = [ perl procps ];
```

```suggestion
  buildInputs = [ procps ];
```
actually do we need this now at all?

---

yep, for the patched shebangs